### PR TITLE
Fix a broken link in the blog post

### PR DESCRIPTION
**Pull request by outsideris**
This link was broken.

### CODE REVIEW

File: locale/en/blog/wg/diag-wg-update-2017-02.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ your experiences getting that info with tools and APIs old and new. [Open an
 issue][] in the Diagnostics WG for discussions or a PR in [nodejs/node][] to
 contribute code.
 
-[Debugging - Getting Started]: http://nodejs.org/en/docs/guides/debugging_getting_started
+[Debugging - Getting Started]: https://nodejs.org/en/docs/guides/debugging_getting_started/
```
Comment (fhemberger) [inline locale/en/blog/wg/diag-wg-update-2017-02.md]: Should be https://nodejs.org/en/docs/guides/debugging-getting-started/

**Review by fhemberger — CHANGES_REQUESTED** [review]

**Comment by outsideris** [comment]
Good point! I updated it.

**Review by gibfahn — APPROVED** [review]

**Comment by fhemberger** [comment]
Great, thank you! Was it only wrong in English or are other languages affected as well?

**Comment by outsideris** [comment]
@fhemberger I found during I'm translating the post as Korean. I will check other languages.

**Pull request merged**